tests/test_integration_download.py

```diff
@@ -26,22 +23,37 @@ def test_download_file():
 
     src_file = "./testdata/file1"
 
-    # download a file
-
-    responses.add(
-        responses.GET,
-        'https://siasky.net/'+SKYLINK,
-        "test\n",
-        status=200
-    )
+    # Download a file.
 
-    dst_file = tempfile.NamedTemporaryFile().name
-    print("Downloading to "+dst_file)
-    client.download_file(dst_file, SKYLINK)
-    if not filecmp.cmp(src_file, dst_file):
-        sys.exit("ERROR: Downloaded file at "+dst_file +
-                 " did not equal uploaded file "+src_file)
+    with tempfile.NamedTemporaryFile() as handle:
+        dst_file = handle.name
+        print("Downloading to "+dst_file)
```

Comment:
Do we need these prints in the integration tests? We usually keep them quiet.

Reply:
`pytest` only displays the stdout output for a test if that test fails, in which case this is helpful for debugging - at least it was for me, so I added it and kept it in.